odbc/2.ibmi.md

```diff
@@ -0,0 +1,105 @@
+# IBM i ODBC Setup
+
+This page will help you set up your ODBC environment for accessing Db2 directly from your IBM i (either your local machine or remote IBM i systems). If you want to connect to your IBM i from a [Linux](./2.linux.md) or [Windows](./2.windows.md) machine, please consult those pages.
+
+## Driver Manager Installation
+
+On IBM i, we will be using unixODBC as our driver manager. Like all open-source software, you need to use yum to install these packages, which will require you to be using IBM 7.2 or greater. If you do not yet have yum installed on your system, [please consult the installation guide here.](https://ibm.biz/ibmi-rpms)
+
+To make sure that unixODBC is available (it should be, if you have yum installed), run `yum list available unixODBC*` to get a print out of all packages matching that name:
+
+```bash
+$ yum list available unixODBC*
+Available Packages
+unixODBC.ppc64                          2.3.4-2                     Artifactory 
+unixODBC-devel.ppc64                    2.3.4-2                     Artifactory 
+```
+
+You should see both `unixODBC` and `unixODBC-devel` packages displayed. To develop your applications using ODBC, you will need to download both both of these packages.
```

Comment:
This whole section about how to use `yum list` to check what's available seems unnecessary. We should not be teaching users how to use yum in this doc, just tell them how to install it (and link to yum docs for those that need it).

Reply:
Additionally, there is no need to install unixODBC separately, since the driver will pull it in automatically when you install it.